DominicTobias/react-image-crop
# Change output units to pixels feature

Hi,

Thanks for this awesome plugin!
I prepared an option to output pixel units instead of percentages.

## Changes in this PR:
- added outputPixelUnits prop to let the user change the units from percentages to pixels,
- updated README,
- updated and configured eslint,
- fixed eslint errors/warnings, this included:
-- reordering some methods inside the React component, the order is proper now, but there is a bug with react/sort-comp plugin, so I had to disable it temporaly. I included a comment in the file,
-- switching from js to jsx extension in entry point file,

My apologies that diff it's not so clear, but I had to make some syntax  changes all over the file to fix linter issues. 

## How to test:
add `outputPixelUnits` prop to the component in `demo/demo.js` file and see the console log output.

Let me know what you think.
